pkg/minikube/cluster/cluster.go

```diff
@@ -48,6 +48,7 @@ import (
 
 var (
 	certs = []string{"apiserver.crt", "apiserver.key"}
+	logsCmd = "docker logs origin"
```

Comment:
+1 :-) Otherwise we could do some name independent things via the container metadata. Eventually I think we need something like this. I think I mentioned that I'd like to build an abstraction layer in Minishift to interact and control the Docker daemon in the VM. Something which is a bit "smarter" than executing plain Docker commands. 

However, if the name of the container is always 'origin' we can go for this approach.

BTW, would there be an issue with exposing the '-f' flag as well to allow following the log?